### DISCUSSION TITLE
Week 12 - Progress Update (Update 9.1.1)

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#243] [General]
Hello everyone!
Here's my status update!

<img width="836" alt="Screenshot 2023-08-06 at 10 23 55 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/55acc0ca-5e80-4cc6-9530-0c5a255b14aa">

Yes, you read that right. Everything is now in the "Done" category. My project is so close to being complete.

I decided against using JS to validate my contact form, because I feel that the `required` tag in HTML already does what I would have implemented in JavaScript.

I want to maybe make a few final tweaks tomorrow, but I am really proud of myself and how far I have come with this project.

Here are a couple of comments I made on other projects:
- https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/242
- https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/241

**Comment by bst3ck** [comment]
The site looks really good. Congrats and well done.

**Reply by coopsand** [reply]
Thanks so much! :)
